BuildResidentialHPXML/tests/build_residential_hpxml_test.rb

```diff
@@ -187,28 +201,40 @@ def test_workflows
       'error-non-integer-ceiling-fan-quantity.xml' => 'Quantity of ceiling fans must be an integer.',
       'error-sfd-conditioned-basement-zero-foundation-height.xml' => "Foundation type of 'ConditionedBasement' cannot have a height of zero.",
       'error-sfd-adiabatic-walls.xml' => 'No adiabatic surfaces can be applied to single-family detached homes.',
-      'error-sfa-ambient.xml' => 'Ambient foundation type for single-family attached or apartment units is not currently supported.',
       'error-mf-conditioned-basement' => 'Conditioned basement/crawlspace foundation type for apartment units is not currently supported.',
       'error-mf-conditioned-crawlspace' => 'Conditioned basement/crawlspace foundation type for apartment units is not currently supported.',
       'error-mf-bottom-crawlspace-zero-foundation-height.xml' => "Foundation type of 'UnventedCrawlspace' cannot have a height of zero.",
       'error-ducts-location-and-areas-not-same-type.xml' => 'Duct location and surface area not both auto or not both specified.',
       'error-second-heating-system-serves-total-heat-load.xml' => 'The fraction of heat load served by the second heating system is 100%.',
       'error-second-heating-system-but-no-primary-heating.xml' => 'A second heating system was specified without a primary heating system.',
       'error-sfa-no-building-num-units.xml' => 'Did not specify the number of units in the building for single-family attached or apartment units.',
-      'error-sfa-above-apartment.xml' => 'Single-family attached buildings cannot be above another unit.',
-      'error-sfa-below-apartment.xml' => 'Single-family attached buildings cannot be below another unit.',
+      'error-sfa-above-apartment.xml' => 'Single-family attached units cannot be above another unit.',
+      'error-sfa-below-apartment.xml' => 'Single-family attached units cannot be below another unit.',
       'error-sfa-all-adiabatic-walls.xml' => 'At least one wall must be set to non-adiabatic.',
       'error-mf-no-building-num-units.xml' => 'Did not specify the number of units in the building for single-family attached or apartment units.',
       'error-mf-all-adiabatic-walls.xml' => 'At least one wall must be set to non-adiabatic.',
       'error-mf-two-stories.xml' => 'Apartment units can only have one above-grade floor.',
+      'error-mf-vented-attic.xml' => 'Apartment units can only have a flat roof or be below another apartment unit.',
```

Comment:
Wait, really? We don't allow a MF unit below an attic? Why is that and how much work would it be to accommodate it? I assume it's reasonably similar to handling a MF unit above a crawlspace or unconditioned basement.

Reply:
Yes, we don't allow it. I believe this is an artifact of us historically only allowing flat roof for multifamily buildings. I opened a separate issue to support this: https://github.com/NREL/OpenStudio-HPXML/issues/965.